.circleci/config.yml

```diff
@@ -52,23 +52,23 @@ jobs:
       - *attach_workspace
       - run:
           name: deploy
-          command: npm run build && sudo npm i -g serverless && sls deploy --stage development
+          command: npm run build && sudo npm i -g serverless@^3 && sls deploy --stage development
```

Comment:
Can also do
``` yaml
command: |
   npm run build
   npm i -g serverless@^3
   sls deploy --stage developmnet
```
to avoid those `&&`s

Reply:
Yes, that would be a bit tidier. I'll leave it for now since it doesn't affect the functionality and the PR itself fixed the issue I was looking into.